### PR TITLE
Fix build that failed due to old patches

### PR DESCRIPTION
**Pull request by azeey**
This also fixes some lintian warnings.

### CODE REVIEW

File: focal/debian/control [hunk 1]
```diff
@@ -61,9 +61,9 @@ Section: libdevel
 Depends: libignition-math6-dev (= ${binary:Version}),
          libeigen3-dev,
          ${misc:Depends}
-Replaces: libignition-math-dev (<< ${binary:Version}),
+Replaces: libignition-math-dev (<< 6.0.0~),
```
Comment (j-rivero) [inline focal/debian/control]: This was set to handle Debian package with is in versions 6.10.0 on jammy. I assume that lintian was complaining, is this correct? Can we leave it as it was or change the 6.0.0 by 6.10.0?

**Review by j-rivero — APPROVED** [review]
If we undo the changes in replaces+breaks, it is good to go if patches apply.

File: focal/debian/control (hunk 1 above)
Comment (azeey) [reply focal/debian/control]: This is the focal control file.

Comment (azeey) [reply focal/debian/control]: The warning I got was https://lintian.debian.org/tags/version-substvar-for-external-package

If you look at `libignition-math6-dev`, the replaces uses `6.0.0`, but the `libignition-math6-eigen3-dev` uses ` ${binary:Version}`. Should they be the same?

Comment (azeey) [reply focal/debian/control]: I've removed this change in https://github.com/gazebo-release/gz-math6-release/pull/22/commits/09634d7925b594a987efa94c59ac64da9e1987cd to unblock this PR, but we should probably address the warning at some point.

**Pull request merged**